src/webrtc_session.ts

```diff
@@ -305,15 +303,15 @@ export class WebRTCSession extends BaseSession {
         }
         return msg.message_id
     }
-    getPayload(): Promise<string | null>{
+    getPayload(): Promise<unknown | void>{
         return new Promise((resolve, reject) => 
             this.sendCTRLMsg({
                 type: "get_payload",
                 args: {}
             }, resolve, reject)
         )
     }
-    setPayload(payload: string): Promise<void>{
+    setPayload(payload): Promise<void>{
```

Comment:
The payload is still a string, right? Why was the type removed?

Reply:
No `Gate.prototype.dump()`  returns `ServerPayload` here https://github.com/tuzig/terminal7/blob/master/src/gate.ts#L593

I'll add the types